pkg/services/object/put/distributed.go

```diff
@@ -224,6 +203,36 @@ func (t *distributedTarget) Close() (oid.ID, error) {
 	return id, nil
 }
 
+func (t *distributedTarget) getShardObjectMetadata() []byte {
```

Comment:
i do not understand what "shard" means here. either we keep in sync with all the previous code, or we describe what "shard" means

Reply:
renamed to "current"